src/test/java/org/folio/rest/api/HoldingsStorageTest.java

```diff
@@ -1542,7 +1542,8 @@ public void cannotCreateAHoldingsWithHRIDFailure()
     final Response response = createCompleted.get(5, TimeUnit.SECONDS);
 
     assertThat(response.getStatusCode(), is(HttpURLConnection.HTTP_INTERNAL_ERROR));
-    assertThat(response.getBody(), isMaximumSequenceValueError("hrid_holdings_seq"));
+    // vertx-pg-client does not return code in message (ONLY message)
+    assertThat(response.getBody(), containsString("hrid_holdings_seq"));
```

Comment:
Does this mean that the only way to tell if this has happened now is by comparing the messages?

Reply:
Yes.

Reply:
@julianladisch Thanks

Reply:
RMB fix is in code review: https://issues.folio.org/browse/RMB-622 = https://github.com/folio-org/raml-module-builder/pull/664

Reply:
And part of RMB 30.0.0 now

Reply:
This is now "fixed" in PR https://github.com/folio-org/mod-inventory-storage/pull/445

Reply:
Can you make the change not here but in the isMaximumSequenceValueError method? If we get the SQLSTATUS code back we only need to update isMaximumSequenceValueError.